x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -0,0 +1,277 @@
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.getLogicalAndPhysicalLabel',
+              {
+                defaultMessage:
+                  'Get logical and physical views of your infrastructure topology.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.breakDownApplicationAndLabel',
+              {
+                defaultMessage:
+                  'Break down application and infrastructure silos for faster root cause detection',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+  ];
+
+  const [selectedId, setSelectedId] = useState<string>();
+  const [hoveredId, setHoveredId] = useState<string>();
+
+  const visibleOption = hoveredId
+    ? options.find((option) => option.id === hoveredId)
+    : selectedId
+    ? options.find((option) => option.id === selectedId)
+    : undefined;
+
+  return (
+    <EuiPanel hasBorder>
+      <TitleWithIcon
+        iconType="indexRollupApp"
+        title={i18n.translate(
+          'xpack.observability_onboarding.experimentalOnboardingFlow.strong.startCollectingYourDataLabel',
+          {
+            defaultMessage:
+              'Start collecting your data by selecting one of the following use cases',
+          }
+        )}
+      />
+      <EuiSpacer size="m" />
+      <EuiFlexGroup css={{ margin: 20 }}>
+        <EuiFlexItem css={{ fontWeight: 'bold' }}>
+          {options.map((option, index) => (
+            <div
+              key={option.id}
+              onMouseEnter={() => setHoveredId(option.id)}
+              onMouseLeave={() => setHoveredId(undefined)}
+            >
+              {index > 0 && <EuiSpacer size="m" />}
+              <EuiCheckableCard
+                id={option.id}
+                label={option.label}
+                checked={selectedId === option.id}
+                onChange={() => setSelectedId(option.id)}
+              />
+            </div>
+          ))}
+        </EuiFlexItem>
+        <EuiFlexItem>
+          {visibleOption && (
+            <EuiPanel color="subdued">
+              <EuiText size="s" color="subdued">
+                {visibleOption.description}
+              </EuiText>
+            </EuiPanel>
+          )}
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      {selectedId && (
+        <>
+          <EuiSpacer />
+          <TitleWithIcon
+            iconType="savedObjectsApp"
+            title={i18n.translate(
+              'xpack.observability_onboarding.experimentalOnboardingFlow.whatTypeOfResourceLabel',
+              {
+                defaultMessage: 'What type of resource are you monitoring?',
+              }
+            )}
+          />
+          <EuiSpacer size="m" />
+
+          {/* Mock integrations grid */}
```

Comment:
This is where https://github.com/elastic/kibana/pull/179301 would hook in, right?

Reply:
Yes, this is just dummy data to show the layout. We will need to integrate the actual integrations grid here.